configuration/config-txt/video.md

```diff
@@ -483,27 +483,31 @@ Setting `hdmi_ignore_hotplug` to `1` pretends that the HDMI hotplug signal is no
 
 ### disable_overscan
 
-Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan).
+Set `disable_overscan` to `1` to disable [overscan](../raspi-config.md#overscan). The default value is `0`.
+
+Disabling overscan is accomplished by setting `overscan_left`, `overscan_right`, `overscan_top` and `overscan_bottom` together to `0`. Any further additional overscan setting such as `overscan_scale` can still be applied.
 
 ### overscan_left
 
-The `overscan_left` command specifies the number of pixels to skip on the left edge of the screen. Increase this value if the text flows off the left edge of the screen; decrease it if there is a black border between the left edge of the screen and the text.
+The `overscan_left` command specifies the number of pixels to skip on the left edge of the screen. Increase this value if the text flows off the left edge of the screen; decrease it if there is a black border between the left edge of the screen and the text. The default value is `48` for HD CEA modes, `32` for SD CEA modes and `0` for DMT modes.
 
```

Comment:
These can now be removed as the information is in the disable_overscan setting.

Reply:
I strongly think the defaults should be kept in each option as per any other option the documentation.

Reply:
No, the default for overscan_left is not 48. It is always zero.
These values get added to the defaults which are set internally by the firmware when ``disable_overscan=0``

Reply:
Ahh I see, so the firmware actually does something like this (very pseudo-code)?
```
overscan_* = 0;
if (disable_overscan == 0) {
  overscan_* = 48, 32, 0; /* according to mode */
}
```

Reply:
Ahhh, so there's "internal default" overscan values (determined either by the display-mode or disable_overscan), which the `overscan_left` etc. values then get added to?
I guess that implies that if the screen-mode is SD CEA, and disable_overscan isn't set, the minimum value for e.g. `overscan_right` is -32 ?

Reply:
@lurch yes

Reply:
Mmm, @popcornmix, so to make things less confusing (for me), is this then what actually happens in the firmware (again very pseudo-code)?
```
internal_overscan_* = 0;
if (disable_overscan == 0) {
  internal_overscan = 48, 32, 0;  /* according to screen-mode */
}
internal_overscan_* += overscan_*;
```

Reply:
Yes